tests/src/test/java/org/eclipse/hono/tests/registry/DeviceRegistrationHttpIT.java

```diff
@@ -179,15 +179,15 @@ public void testAddDeviceFailsForMissingContentType(final TestContext ctx) {
     }
 
     /**
-     * Verifies that a device cannot be registered if the request
+     * Verifies that a device can be registered if the request
      * does not contain a body.
      * 
      * @param ctx The vert.x test context
      */
     @Test
-    public void testAddDeviceFailsForMissingBody(final TestContext ctx) {
+    public void testAddDeviceSucceedsForMissingBody(final TestContext ctx) {
```

Comment:
I would suggest to name this *testAddDeviceSucceedsForEmptyBody* as the body is not missing in this case, is it?

Reply:
Indeed, in this case, the body seems to be an empty JsonObject.. Now I don't understand why the test fails, as it should work with an empty JsonObject, as demonstrated in my other PR..

Reply:
from the perspective of the *registerDevice* method that is correct. However, we are not testing the client here but the service implementation and from the point of view of the service implementation the body is not *missing* but simply *empty*. The same holds true for the test that you have added ...

Reply:
I updated the names